core/communication/port-authority/src/util.ts

```diff
@@ -47,12 +45,10 @@ export async function rethrowPromise<T>(t: Try<T>): Promise<T> {
 export async function recoverPromise<T>(p: Promise<T>): Promise<Try<T>> {
     try {
```

Comment:
I did not understood at first why it is needed to have Try in Promise, as Promise already should have success/error mechanism.

But looking at the usages of this function, this seems to added to build the network request body, here -> https://github.com/polypoly-eu/polyPod/blob/ec169bf17049ef5487cb729654a9f4638c4cabe5/core/communication/port-authority/src/node.ts#L86

This also begs the question if it is ok to remove the `tag` field, it seems to be part of the network request contract.

Reply:
Let me check that.

Reply:
OK, I've not removed the functionality, just the use of two strings as specific tag values. I've used the attribute `err` to signal the same.